tests/ui/scripts/generate_fxa_email.py

```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def create_fxa_email():
+    id = uuid.uuid1()
+    email = f"uitest-{id}@restmail.net"
+    print(f"{email}")
```

Comment:
Heh, you could do this in less lines, but meh.

Reply:
Challenge excepted